beetmoverscript/docker.d/worker.yml

```diff
@@ -433,7 +433,7 @@ clouds:
               nightly_components: 'maven-nightly-s3-upload-bucket-d4zm9oo354qe'
           nightly:
             fail_task_on_error: True
-            enabled: True
+            enabled: False
```

Comment:
I see `nightly` is getting disabled for prod but not fake-prod and dev - is that an oversight?

Reply:
It was, thanks.  I think I got distracted wondering whether it was time to get rid of the TODO.